package.json

```diff
@@ -16,93 +16,49 @@
-    "preversion": "rimraf node_modules && npm install && npm run test",
-    "version": "npm run build && git add dist",
-    "postversion": "git push && git push --tags && npm run update-gh-pages && echo Tag v$npm_package_version created. Run \"npm publish\" to publish \"$npm_package_name\" on npmjs.com."
+    "version": "npm run build && git add dist"
   },
   "devDependencies": {
-    "autoprefixer": "^9.7.1",
+    "@codeceptjs/configure": "^0.5.2",
+    "@vue/cli-plugin-babel": "~4.3.0",
+    "@vue/cli-plugin-eslint": "~4.3.0",
+    "@vue/cli-plugin-unit-jest": "~4.3.0",
+    "@vue/cli-service": "~4.3.0",
+    "@vue/eslint-config-airbnb": "^5.0.2",
+    "@vue/test-utils": "1.0.0-beta.31",
+    "@wdio/selenium-standalone-service": "^5.0.0",
     "axios": "^0.19.0",
-    "babel-core": "^6.26.3",
-    "babel-eslint": "^8.2.6",
-    "babel-loader": "^7.1.5",
-    "babel-plugin-istanbul": "^4.1.6",
-    "babel-plugin-transform-runtime": "^6.23.0",
-    "babel-preset-env": "^1.7.0",
-    "babel-preset-stage-2": "^6.24.1",
-    "babel-register": "^6.26.0",
-    "banner-webpack-plugin": "^0.2.3",
-    "chai": "^4.2.0",
-    "chalk": "^2.4.2",
-    "codeceptjs": "^2.3.5",
-    "copy-webpack-plugin": "^4.6.0",
-    "cross-env": "^6.0.3",
+    "babel-eslint": "^10.1.0",
+    "codeceptjs": "^2.3.6",
+    "codeceptjs-webdriverio": "^1.1.0",
+    "core-js": "^3.6.4",
     "cross-spawn": "^7.0.1",
-    "css-loader": "^0.28.11",
-    "eslint": "^4.19.1",
-    "eslint-config-airbnb-base": "^12.1.0",
-    "eslint-friendly-formatter": "^3.0.0",
-    "eslint-import-resolver-webpack": "^0.8.4",
-    "eslint-loader": "^1.9.0",
-    "eslint-plugin-html": "^4.0.6",
-    "eslint-plugin-import": "^2.18.2",
-    "eventsource-polyfill": "^0.9.6",
-    "extract-text-webpack-plugin": "^3.0.2",
-    "file-loader": "^4.2.0",
-    "friendly-errors-webpack-plugin": "^1.7.0",
-    "function-bind": "^1.1.1",
-    "html-webpack-plugin": "^3.2.0",
-    "import-glob-loader": "^1.1.0",
-    "inject-loader": "^4.0.1",
-    "karma": "^4.4.1",
-    "karma-coverage": "^2.0.1",
-    "karma-mocha": "^1.3.0",
-    "karma-phantomjs-launcher": "^1.0.4",
-    "karma-sinon-chai": "^2.0.2",
-    "karma-sourcemap-loader": "^0.3.7",
-    "karma-spec-reporter": "0.0.32",
-    "karma-webpack": "^2.0.13",
+    "eslint": "^6.7.2",
+    "eslint-plugin-import": "^2.20.2",
+    "eslint-plugin-vue": "^6.2.2",
+    "jest-canvas-mock": "^2.2.0",
     "material-design-icons-iconfont": "^5.0.1",
-    "mocha": "^4.1.0",
-    "nightmare": "^3.0.2",
-    "nightmare-upload": "^0.1.1",
-    "node-notifier": "^6.0.0",
-    "node-sass": "^4.13.0",
-    "optimize-css-assets-webpack-plugin": "^3.2.1",
-    "ora": "^4.0.2",
-    "phantomjs-polyfill-find": "0.0.1",
-    "phantomjs-prebuilt": "^2.1.16",
-    "portfinder": "^1.0.25",
-    "postcss-import": "^12.0.1",
-    "postcss-loader": "^3.0.0",
+    "node-sass": "^4.12.0",
```

Comment:
What do you think about replacing `node-sass` with just `sass` that has no native dependencies?